packages/store-ui/src/atoms/Bullets/Bullets.tsx

```diff
@@ -0,0 +1,64 @@
+import type { MouseEvent } from 'react'
+import React from 'react'
+
+import Button from '../Button'
+
+export interface BulletsProps {
+  /**
+   * Number of bullets that should be rendered.
+   */
+  totalQuantity: number
+  /**
+   * The currently active bullet (zero-indexed).
+   */
+  activeBullet: number
+  /**
+   * Event handler for clicks on each bullet. The handler will be called with
+   * the index of the bullet that received the click.
+   */
+  onClick: (e: MouseEvent, bulletIdx: number) => void
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  /**
+   * ID to find each `<li>` rendered by this component in testing tools
+   * (e.g.: cypress, testing-library, and jest).
+   */
+  listItemTestId?: string
+}
+
+function Bullets({
+  totalQuantity,
+  activeBullet,
+  onClick,
+  testId = 'store-bullets',
+  listItemTestId = 'bullet-item',
```

Comment:
Doesn't it look like we are making a molecule become an atom? For me it looks we have an atom and a molecule in here. The atom is the `Bullet` and the `Bullets` is the molecule that ueses this Bullet atom. This way we can have one single prop TestId. What do you think? Is this overcomplicating?

Reply:
But Bullet is an Atom that only used by Bullets? Do not make sense to me to create an atom just for it.

Reply:
Yeah, I'm thinking the same as Emerson. I do think it makes sense for this whole thing to be an atom since this is a minimum functional arrangement of these HTML elements. A `Bullet` would just be a `<li>` with a button inside, that would not fit into other contexts (except for maybe another list component). If we do have an Atom such as `Bullet`, I wouldn't call it that, but something like `list-item`... but then again, I don't know how useful that would turn out to be

Reply:
But another scenario that got me confused. What if we were talking about a `Menu`? It doesn't feel right to me to make an `Atom`. How would that fit in here? Any ideas @emersonlaurentino ?

Reply:
Like that

![image](https://user-images.githubusercontent.com/10627086/123125745-2d0f8a80-d41f-11eb-876a-01ed10ff0fc0.png)

The Menu Item does not need is an Atom, the atom can be a button or anchor component, but into the Menu (molecule) these components are MenuItem.

Reply:
Oooh! Okay, that makes sense!
But in that case, I would argue that `Bullets` is a molecule, right?

Reply:
We don't need to create a `Bullet` atom, but the `Bullets` components would be a molecule that uses the `<Button>` atom (which it already does)

Reply:
Exactly!